tiledb/sm/array/array.h

```diff
@@ -1214,6 +1215,17 @@ class Array {
   void set_array_closed();
 };
 
+/**
+ * Loads an enumeration into a schema.
+ * Used to implement `tiledb_array_schema_get_enumeration*` APIs.
+ *
+ * @param ctx
+ * @param enmr_name the requested enumeration
+ * @param schema the target schema
+ */
+void load_enumeration_into_schema(
```

Comment:
Could this maybe go into `array_schema_operations.cc`, or did you also run into problems there with dependencies? At a glance it seems like it could work

Reply:
I believe it won't work there either.  I am pretty new to cmake but my impression is that all the `array_schema/*.cc` files are compiled into the same "object library", hence putting this code in `array_schema_operations.cc` would also have the circular dependency between `array` and `array_schema`